test/parallel/test-trace-events-http.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const cp = require('child_process');
+const fs = require('fs');
+const path = require('path');
+const tmpdir = require('../common/tmpdir');
+
+const CODE = `
+  const http = require('http');
+  const server = http.createServer((req, res) => {
+    res.end('ok');
+    server.close();
+  }).listen(0, () => {
+    http.get({port: server.address().port});
+  });
+`;
+
+tmpdir.refresh();
+const FILE_NAME = path.join(tmpdir.path, 'node_trace.1.log');
+
+const proc = cp.spawn(process.execPath,
+                      [ '--trace-events-enabled',
+                        '--trace-event-categories', 'node.http',
+                        '-e', CODE ],
+                      { cwd: tmpdir.path });
+
+proc.once('exit', common.mustCall(() => {
+  assert(fs.existsSync(FILE_NAME));
+  fs.readFile(FILE_NAME, common.mustCall((err, data) => {
+    assert(!err);
+    const traces = JSON.parse(data.toString()).traceEvents;
```

Comment:
JSON.parse can throw an error, I think a try catch would improve the error stack, if in the future it throws.

Reply:
If error is threw, it means the test fails. So i think we do not need to catch it ?